policyengine_us/parameters/gov/states/az/hhs/tanf/eligibility/income/earned/flat.yaml

```diff
@@ -0,0 +1,11 @@
+description: Arizona disregards this amount of earned income under the cash assistance program.
+metadata:
+  unit: currency-USD
+  label: Arizona Cash Assistance flat earned income disregard
+  period: year
+  reference:
```

Comment:
can you find a primary source?

Reply:
@liwenxin491 could you find the numbers here and add it as a reference:
https://dbmefaapolicy.azdes.gov/index.html#page/FAA1/The_Cash_Assistance_(CA)_Program.html